content/authentication/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Chapter 19: Authentication"
+date: 2021-10-01T09:28:27-05:00
+draft: false
+weight: 19
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Sally Steuterman # to be set by the page reviewer
+reviewerGitHub: gildedgardenia # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Learning Objectives
+
+After completing this chapter, you should be able to do the following:
+1. Understand the basic concepts of authentication
+1. Use request filters to enable basic authentication in a Spring app
+1. Explain what two-way encryption is, and why it is not ideal for storing sensitive user data, such as passwords
+1. Explain what a one-way hash is, and why it presents greater security for storing sensitive user data
+1. Describe the characteristics of a good one-way hash function
+1. Write code that uses a provided hash function to hash sensitive data
+1. Describe how one-way hashes allow for passwords to be compared
+1. Describe the general authentication flow for a web app
+
+## Key Terminology
```

Comment:
Please add explanations and section names here!

Reply:
updated key terms section. Added next-steps.